### PR TITLE
Add option to use the test config for build 

### PR DESCRIPTION
**Pull request by nazarhussain**
Add option to use test configuration and test genesis block to generate the build

### CODE REVIEW

**Review by Nazgolze — APPROVED** [review]
This looks good to me

**Review by Nazgolze — DISMISSED** [review]
The commits need to be signed though

File: vars/liskBuild.groovy [hunk 1]
```diff
@@ -32,10 +59,10 @@ def call(Map params) {
...
 		git url: "https://github.com/LiskHQ/lisk-build.git", branch: liskBuildBranch
 		sh """
-		cp "../${liskDir}/release/${liskReleaseName}" "src/"
+		cp "${liskDir}/release/${liskReleaseName}" "src/"
 		bash build.sh -n "${buildNetwork}" -v "${liskVersion}"
 		"""
 	}
 
-	return [version: liskVersion, latestVersion: latestVersion, network: buildNetwork, file: "${liskBuildDir}/release/lisk-${liskVersion}-Linux-x86_64.tar.gz"]
+	return [version: liskVersion, latestVersion: latestVersion, network: buildNetwork, file: "${liskBuildDir}/release/lisk-Linux-x86_64.tar.gz"]
```
Comment (abelboldu) [inline vars/liskBuild.groovy]: The output of lisk-build is versioned I think @Nazgolze

Comment (Nazgolze) [reply vars/liskBuild.groovy]: Ah - the nightly builds

Comment (nazarhussain) [reply vars/liskBuild.groovy]: @Nazgolze How to resolve this issue? I am in favor to have version there. But with  version `liskInstall.sh` script does not work.

Comment (Nazgolze) [reply vars/liskBuild.groovy]: @nazarhussain @abelboldu Technically, we don't need to use what gets returned in the jenkins migration job. lisk-build is always going to create "${liskBuildDir}/release/lisk-Linux-x86_64.tar.gz".

So we can have it still return "${liskBuildDir}/release/lisk-${liskVersion}-Linux-x86_64.tar.gz" for the nightly builds

**Review by abelboldu — DISMISSED** [review]

File: vars/liskBuild.groovy (hunk 1 above)
Comment (Nazgolze) [reply vars/liskBuild.groovy]: after talking with @nazarhussain, it's easiest to add a nonVersionedFile field

File: vars/liskBuild.groovy [hunk 2]
```diff
@@ -27,15 +54,15 @@ def call(Map params) {
...
 		git url: "https://github.com/LiskHQ/lisk-build.git", branch: liskBuildBranch
 		sh """
-		cp "../${liskDir}/release/${liskReleaseName}" "src/"
+		cp "${liskDir}/release/${liskReleaseName}" "src/"
 		bash build.sh -n "${buildNetwork}" -v "${liskVersion}"
 		"""
 	}
 
-	return [version: liskVersion, latestVersion: latestVersion, network: buildNetwork, file: "${liskBuildDir}/release/lisk-${liskVersion}-Linux-x86_64.tar.gz"]
+	return [version: liskVersion, latestVersion: latestVersion, network: buildNetwork, file: "${liskBuildDir}/release/lisk-${liskVersion}-Linux-x86_64.tar.gz", unversionedFile: "${liskBuildDir}/release/lisk-${liskVersion}-Linux-x86_64.tar.gz"]
```
Comment (Nazgolze) [inline vars/liskBuild.groovy]: `unversionedFile: "${liskBuildDir}/release/lisk-${liskVersion}-Linux-x86_64.tar.gz"`

This should be either: 

- `unversionedFile: "${liskBuildDir}/release/lisk-Linux-x86_64.tar.gz"`
- `nonVersionedFile: "${liskBuildDir}/release/lisk-Linux-x86_64.tar.gz"`

depending on how the Jenkins job is going to get at it

**Review by Nazgolze — CHANGES_REQUESTED** [review]

**Review by Nazgolze — APPROVED** [review]

**Comment by abelboldu** [comment]
Please rebase first

**Pull request merged**